README.adoc

```diff
@@ -183,8 +186,8 @@ You will create an endpoint test to test the basic functionality of the `gateway
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `JobGatewayEndpointTest` class.#
-`gateway/src/test/java/it/io/openliberty/guides/gateway/JobGatewayEndpointTest.java`
+#Create the `GatewayJobEndpointIT` class.#
```

Comment:
What's the reason for renaming the test class with "IT"? We haven't been using this convention.